### DISCUSSION TITLE
How do I embed Youtube automatically generated subtitles

### DISCUSSION THREAD

**Discussion by Booms1981** [JunkFood02/Seal#717] [General]
Tried a few options like --write-auto-subs but that's not working. It's autogenerated Spanish if that makes any difference?

**Comment by Marco-9456** [comment]
https://user-images.githubusercontent.com/41828058/233779625-cc51617a-f36a-4faa-aaf4-1183c62bcdd1.mp4

**Reply by Booms1981** [reply]
> Screen_Recording_20230422_124804_Seal.mp4

Perfect - thanks
